tests/phpunit/tests/formatting/maybeHashHexColor.php

```diff
@@ -7,12 +7,15 @@
  *
  * @covers ::maybe_hash_hex_color
  */
-class Tests_formating_maybeHashHexColor extends WP_UnitTestCase {
+class Tests_Formatting_maybeHashHexColor extends WP_UnitTestCase {
 
 	/**
 	 * @ticket 60272
 	 *
 	 * @dataProvider date_sanitize_hex_color_no_hash
+	 *
+	 * @param string $color    Color.
+	 * @param string $expected Expected.
```

Comment:
The data provider docblock should be:

```php
/**
 * Data provider.
 *
 * @return array[]
 */
```

🔢 May apply to multiple files in the PR.